tryItOut("try{o}catch(__count__){const __count__=0}")